runtime/js/99_main.js

```diff
@@ -216,6 +216,9 @@ delete Object.prototype.__proto__;
       runtimeOptions.v8Version,
       runtimeOptions.tsVersion,
     );
+    if (runtimeOptions.unstableFlag) {
+      internals.enableTestSteps();
```

Comment:
It might be better if we instead stored whether we're running with the unstable flag. I wasn't sure where that could go though.